domains/DataCollection/Forms/IdentificationForm/index.js

```diff
@@ -42,7 +44,7 @@ const IdentificationForm = ({ navigation, scrollViewScroll, setScrollViewScroll
     return () => {
       clearInterval(interval);
     };
-  }, []);
+  }, [backgroundPostPatient, clearInterval]);
```

Comment:
I think there's a dependencies issue with some of the background processes but I don't know what it is yet so I just created a bug ticket for us to address next sprint @jamccomb92